src/Serilog/Settings/KeyValuePairs/SettingValueConversions.cs

```diff
@@ -29,6 +35,22 @@ class SettingValueConversions
 
         public static object ConvertToType(string value, Type toType)
         {
+            if (TryParseStaticMemberAccessor(value, out var accessorTypeName, out var memberName))
+            {
+                var accessorType = Type.GetType(accessorTypeName, throwOnError: true);
+                var publicStaticPropertyInfo = accessorType.GetTypeInfo().DeclaredProperties
+                    .Where(x => x.Name == memberName)
```

Comment:
Perhaps we could also add support for the public static fields at the same time?

Reply:
I have mixed feelings about accessing arbitrary properties, but I guess if they are `public`, they are part of the public API so technically accessible. I'll include public properties as well